src/readit/details/comments/commentItem.js

```diff
@@ -45,13 +46,13 @@ const CommentItem = (props)=> {
             <div className="row">
                 <div className="col-2 text-center float-left">
                     <img style={{height:"80px", width:"80px"}} src={avatar} className="col-12 rounded-circle"/>
-                    <div className="overflow-hidden"><a href={`/readit/profile/${comment.username}`}> {comment.username}</a></div>
+                    <div className="overflow-hidden"><a href={`/readit/profile/username/${comment.username}`}> {comment.username}</a></div>
```

Comment:
This needs to be changed since we are not looking at profiles via username anymore, we require ID